### DISCUSSION TITLE
need help engine in ahk.window.Window.from_pid

### DISCUSSION THREAD

**Discussion by pp7sec** [spyoungtech/ahk#157] [Q&A] [answered]
```
from ahk import AHK
from ahk.window import Window

ahk = AHK()
win = Window.from_pid(pid='5392')
```
>> TypeError: from_pid() missing 1 required positional argument: 'engine'

What is engine ?
Thanks

**Comment by spyoungtech** [comment, marked as answer]
engine should be the AHK instance:

```python
ahk = AHK()
win = Window.from_pid(ahk, pid='5392')
```

It seems there's an error in the README where the argument is omitted. ~~I'll fix this soon!~~ Fixed

EDIT:

You can also use `win_get` instead of using `Window.from_pid`

```python
ahk = AHK()
win = ahk.win_get(title='ahk_pid 20366')
```
